docs/antrea-network-policy.md

```diff
@@ -459,17 +532,30 @@ prohibited` code for ICMP, UDP and SCTP request, or an explicit reject response
 TCP request, instead of timeout. A "Pass" rule, on the other hand, skips this packet
 for further ACNP rule evaluations (all ACNP rules that has lower priority than the
 current "Pass" rule will be skipped, except for the Baseline Tier rules), and delegates
-the decision to developer created namespaced NetworkPolicies. If no NetworkPolicy matches
-this traffic, then the Baseline Tier rules will still be matched against. Note that the
-"Pass" action does not make sense when configured in Baseline Tier ACNP rules, and such
-configurations will be rejected by the admission controller.
+the decision to developer created namespaced NetworkPolicies. If no NetworkPolicy matches this traffic,
+then the Baseline Tier rules will still be matched against. Note that the "Pass" action does not
+make sense when configured in Baseline Tier ACNP rules, and such configurations will be rejected
+by the admission controller. Note: multicast NetworkPolicy does not support "Pass" and "Reject" action.
 
 **ingress**: Each ClusterNetworkPolicy may consist of zero or more ordered set of
 ingress rules. Under `ports`, the optional field `endPort` can only be set when a
 numerical `port` is set to represent a range of ports from `port` to `endPort` inclusive.
 `protocols` defines additional protocols that are not supported by `ports`. Currently, only
-ICMP protocol is under `protocols`. `icmpType` and `icmpCode` could be used to specify the ICMP
-traffic that this rule matches.
+ICMP protocol and IGMP are under `protocols`. For `ICMP` protocol, `icmpType` and `icmpCode`
+could be used to specify the ICMP traffic that this rule matches. And for `IGMP` protocol,
+`igmpType` and `groupAddress` can be used to specify the IGMP traffic that this rule matches.
+Currently, only IGMP query is valid in ingress.   
+Valid `igmpType` is:
+
+message type | value
+-- | --
+Membership Query | 0x11
+
+IGMP v3 supports query specific group addresses, which are encoded in IGMP 
```

Comment:
IGMP query group address only works on `224.0.0.1`. As for Group-Specific IGMP query, which encodes the target group in the protocol, it is not supported yet because OVS can not recognize the address. Currently, ingress rules for multicast traffic are not supported.

Reply:
done